akka-stream-tests/src/test/scala/akka/stream/actor/ActorSubscriberSpec.scala

```diff
@@ -25,13 +25,13 @@ object ActorSubscriberSpec {
     override val requestStrategy = ZeroRequestStrategy
 
     def receive = {
-      case next @ OnNext(elem)   ⇒ probe ! next
-      case complete @ OnComplete ⇒ probe ! complete
-      case err @ OnError(cause)  ⇒ probe ! err
-      case "ready"               ⇒ request(elements = 2)
-      case "boom"                ⇒ throw new RuntimeException("boom") with NoStackTrace
-      case "requestAndCancel"    ⇒ { request(1); cancel() }
-      case "cancel"              ⇒ cancel()
+      case next @ OnNext(elem)  ⇒ probe ! next
+      case OnComplete           ⇒ probe ! OnComplete
```

Comment:
May I ask why this change? (given that the one before and after extracts values that it doesn't use)